src/pages/components/MinuteSelector.tsx

```diff
@@ -1,23 +1,36 @@
 import React from 'react'
 import { TEXTS } from 'src/resources/texts'
 import { TextField } from '@mui/material'
+import ClearButton from './ClearButton'
 
 type MinuteSelectorProps = {
   num: number
   setNum: (num: string) => void
 }
 
 const MinuteSelector = ({ num, setNum }: MinuteSelectorProps) => {
+  const handleClearInput = () => {
+    setNum('0')
+  }
+
   return (
     <TextField
-      sx={{ width: '100%' }}
+      sx={{
+        width: '100%',
+        '&:hover .clearIndicatorDirty , & .Mui-focused .clearIndicatorDirty ': {
+          visibility: num ? 'visible' : 'hidden',
+        },
+      }}
```

Comment:
could you please move it to css file?

Reply:
yes, I will try to do it even though it contains some logic and reference to the mui component classes.

Reply:
You can use conditional 'className'